### ISSUE TITLE
does it works well with vue3?

### ISSUE THREAD

**Issue by bokeer** [egoist/vue-monaco#156]
Uncaught TypeError: h is not a function

**Comment by mustafam-globant** [comment]
From Docs-
In 3.x, h is now globally imported instead of being automatically passed as an argument.
https://v3.vuejs.org/guide/migration/render-function-api.html#render-function-argument

Implementing this in MonacoEditor.js solved the issue for me.

**Comment by Ph0enixKM** [comment]
Yes, you just have to write this line under _MonacoEditor_ import:
```js
MonacoEditor.render = () => h('div')
```

**Comment by IOSDongba** [comment]
2.x语法
在2.x中，该render函数将自动接收该h函数（这是的常规别名createElement）作为参数：

// Vue 2 Render Function Example
export default {
  render(h) {
    return h('div')
  }
}
＃3.x语法
在3.x中，h现在已全局导入，而不是自动作为参数传递。

// Vue 3 Render Function Example
import { h } from 'vue'

export default {
  render() {
    return h('div')
  }
}

**Comment by mike-seekwell** [comment]
```
import MonacoEditor from 'vue-monaco'
import { h } from 'vue'
MonacoEditor.render = () => h('div')
```
